src/compiler/diagnosticMessages.json

```diff
@@ -3511,6 +3511,14 @@
         "category": "Error",
         "code": 8021
     },
+    "Generic type '{0}' requires '{1}' type arguments; provide these with an '@augments' or '@extends' tag.": {

```

Comment:
I'd choose one of `@augments` or `@extends` for the error, preferably the one that is in the source if it is already there.

Reply:
There probably won't be one already there; we might just recommend `@extends` since that's the more intuitive name.